.github/workflows/cairo-update-check.yml

```diff
@@ -0,0 +1,49 @@
+name: Cairo Update Check
+
+on:
+  workflow_dispatch:
+    inputs:
+      slack:
+        description: "Send Slack notification on failure"
+        type: boolean
+        default: true
+  schedule:
+    - cron: '0 0 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
```

Comment:
Why do we need it?

Reply:
This mostly came by copy-pasting, but in general for this workflow always the last run represents the most fresh state so we will never be interested in old runs, so why not?